### PR TITLE
ci: integrate OpenShift setup and test execution into the CI process 

### PR DESCRIPTION
**Pull request by chavafg**
This change will allow to setup Openshift environment and execute tests 
through the CI if envar OPENSHIFT_CI=true. The purpose of this change is
to make minimal changes in our jenkins setup.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,6 +39,13 @@ metrics:
 integration: ginkgo
 	./ginkgo ./integration/docker/ -- -timeout ${TIMEOUT}
 
+openshift:
+	cd integration/openshift
+	./init.sh
+	# Sleep 30 seconds to let the master and node initialize correctly
+	sleep 30
```
Comment (jodh-intel) [inline Makefile]: This looks a bit odd and potentially fragile. Is there no way to check when the nodes have initialised fully (a command to run, particular log message(s) to look for, etc)?

Comment (chavafg) [reply Makefile]: not sure, but will take a look

Comment (chavafg) [reply Makefile]: changed the approach to verify if the node is up every 5 secs until a maximum of 30s.

**Comment by jodh-intel** [comment]
lgtm

**Comment by devimc** [comment]
lgtm

**Pull request merged**